src/src.pro

```diff
@@ -28,6 +28,11 @@ nogui {
         DEFINES += QBT_STATIC_QT
         QTPLUGIN += qico
     }
+    win32: {
```

Comment:
You shouldn't use both `:` and `{}`. The first is for single line statements, the second one is for multiline.